src/themes/bulma/bootstrap-table-bulma.scss

```diff
@@ -25,17 +25,22 @@
       margin-right: 0;
     }
 
-    .keep-open.button.dropdown {
+    .button.dropdown {
       padding: 0;
-      margin: 0;
       border: 0;
-
+      
```

Comment:
Should remove the spaces here.

Reply:
Done. https://github.com/wenzhixin/bootstrap-table/pull/4236/commits/64786dc59930a5e8e069a576b611b6ce1545d99b